library/src/morpheus-master/js/src/sakai.morpheus.usernav.js

```diff
@@ -332,3 +332,12 @@ $PBJQ(document).ready( function(){
 
   $PBJQ("body").on("minimise-tool", e => minimiseTool());
 });
+
+/////////////////////////////////////////////////
+// Add become user to body as a class
+////////////////////////////////////////////////
+$PBJQ(document).ready( function(){
+    if ( $PBJQ('#loginLink1').find('.Mrphs-login-Message').text().startsWith("Return") ){
```

Comment:
This seems a bit off. This would be an internationalised string and may not be "Return".

Reply:
I'll fix this with another commit.